Automattic/jetpack
# Social: Fix logic issue in store for enhanced publishig

Fixes p1693215729811629-slack-C02JJ910CNL

There was an issue in the Social store, that resulted in the `isEnhancedPublishingEnabled` flag to be flipped. This fixes that.

### Other information:


## Jetpack product discussion


p1693215729811629-slack-C02JJ910CNL
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Create a new JN site with the Jetpack Complete plan and Jetpack Social
* Connect the page to your account and open the Social admin page
* You should see the Instagram notice, but it should not nudge you for an upgrade:
<img width="1177" alt="CleanShot 2023-08-28 at 12 24 40 png" src="https://github.com/Automattic/jetpack/assets/36671565/248d01f5-d7c0-481b-a5ba-84bfcb93765c">

---

* Check the same without this change, it should have the nudging part in it:
![image](https://github.com/Automattic/jetpack/assets/36671565/bdfdfa45-eabe-4d2c-aba9-2e795ce1fc55)
